.github/workflows/run_ec2_pcluster.yaml

```diff
@@ -20,7 +20,9 @@ jobs:
 
     steps:
       - name: cleanup
-        if: ${{ github.event_name == 'schedule' }}
+        # DH* TO ALLOW FOR TESTING PARALLEL TO OTHER PRS
```

Comment:
Note. Revert this before merging (needed so that we can test different PRs with spack-stack-1.5.0 and 1.5.1)